test/test-eta.js

```diff
@@ -0,0 +1,843 @@
+'use strict'
+
+const t = require('tap')
+const test = t.test
+const sget = require('simple-get').concat
+const Fastify = require('fastify')
+const fs = require('fs')
+const path = require('path')
+const minifier = require('html-minifier')
+const minifierOpts = {
+  removeComments: true,
+  removeCommentsFromCDATA: true,
+  collapseWhitespace: true,
+  collapseBooleanAttributes: true,
+  removeAttributeQuotes: true,
+  removeEmptyAttributes: true
+}
+
+test('reply.view with eta engine and custom templates folder', t => {
+  t.plan(6)
+  const fastify = Fastify()
+  const eta = require('eta')
```

Comment:
move all the `require('eta')` to the top.

Reply:
Just fixed it :smiley: